src/fairness/weighted_tree/test/load_accounting_db.cpp

```diff
@@ -112,50 +156,42 @@ std::shared_ptr<weighted_tree_node_t> get_sub_banks (
                             sqlite3 *DB,
                             const std::string &bank_name,
                             std::shared_ptr<weighted_tree_node_t> parent_bank,
-                            sqlite3_stmt *b_select_shares_stmt,
-                            sqlite3_stmt *b_select_sub_banks_stmt,
-                            sqlite3_stmt *b_select_associations_stmt)
+                            sqlite3_stmt *c_shrs,
+                            sqlite3_stmt *c_sub_banks,
+                            sqlite3_stmt *c_assoc)
 {
 
     std::shared_ptr<weighted_tree_node_t> node = nullptr;
     std::shared_ptr<weighted_tree_node_t> user_node = nullptr;
```

Comment:
I believe `user_node` is used incorrectly. Passing `nullptr` to `add_assoc` later is not used within that function. You should just remove this variable. I will leave another comment to the callosities of `add_assoc`.